SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -892,6 +893,9 @@ void FSpatialGDKEditorToolbarModule::VerifyAndStartDeployment(FString ForceSnaps
 void FSpatialGDKEditorToolbarModule::StartLocalSpatialDeploymentButtonClicked()
 {
 	VerifyAndStartDeployment();
+	const USpatialGDKEditorSettings* SpatialGDKEditorSettings = GetDefault<USpatialGDKEditorSettings>();
+	const FString RuntimeVersion = SpatialGDKEditorSettings->GetSelectedRuntimeVariantVersion().GetVersionForLocal();
+	FSpatialPackageManager::FetchRuntimeBinary(RuntimeVersion);
```

Comment:
Shouldn't this be before starting the deployment?

Reply:
Hmm, the runtime variant is dynamic (can be change in editor) no? When else could we do this? Perhaps on editor load and changing the setting?

Reply:
I don't think this will be the permanent place for fetching the runtime binary; but even now we're okay with the Deployment being successful independently from these binaries